fedimint-server/src/consensus/mod.rs

```diff
@@ -148,14 +149,26 @@ impl FedimintConsensus {
                                 .await
                                 .expect("Setting transaction savepoint failed");
 
+                            if let  ConsensusItem::Transaction(ref transaction) = consensus_item {
+                                if accepted_txes.contains(&transaction.tx_hash()) {
+                                    continue;
+                                }
+                            }
+
```

Comment:
Just skipping the transaction without making a decision will be a bug once we migrate to AlephBFT (correct me if I'm wrong @joschisan). So ideally we'd instead just make the insert operation into `rejected_txs` dependent on if the rejection happened due to invalidity or double-inlusion. The concept of rejected transactions will disappear with AlephBFT anyway since only accepted ones will be persisted in the on-disk log.

https://github.com/fedimint/fedimint/blob/6603c8bc2c6e9e616d89f26984af6560b4e75c31/fedimint-server/src/consensus/mod.rs#L177-L179